app/grandchallenge/products/models.py

```diff
@@ -152,3 +152,8 @@ class Meta:
 
     def get_absolute_url(self):
         return reverse("products:product-detail", kwargs={"slug": self.slug})
+
+
+class ProjectAirFiles(models.Model):
+    title = models.CharField(max_length=150)
+    study_file = models.FileField(upload_to=get_pdf_path)
```

Comment:
This will make them all publicaly available, is that ok?

Reply:
they should be downloadable files, so yes.